src/test/java/com/verhas/licensor/TestLicenseClass.java

```diff
@@ -53,6 +53,14 @@ public void tearDown() throws IOException {
         new File(dumpFile2).delete();
     }
 
+    private int getOffset() {
+        if (System.getProperty("os.name").startsWith("Windows")){
```

Comment:
What about OpenVMS? Or some other OS that Java runs on and you did not think about? Also: 2 and 1 in this case are magic constants even if tools like PMD, Checkstyle think they are not.

Why not `System.getProperty("line.separator").length()` ? More general and tells immediately what you want to do. If speed is concern store it in a 

`private static final int nrLineSepChars = System.getProperty("line.separator").length();`

variable and forget the whole method.